pytools/__init__.py

```diff
@@ -1486,13 +1486,22 @@ def __call__(self, *args, **kwargs):
 class Table:
     """An ASCII table generator.
 
+    :arg alignments: List of alignments of each column ('l', 'c', or 'r',
+        for left, center, and right alignment, respectively). Currently only used
+        by the `github_markdown` output formatter. Columns which have no alignment
+        specifier will use the last specified alignment. For example, with
+        `alignments=['l', 'r']`, the third and all following columns will use 'r'
+        alignment.
+
     .. automethod:: add_row
     .. automethod:: __str__
     .. automethod:: latex
+    .. automethod:: github_markdown
     """
 
-    def __init__(self):
+    def __init__(self, alignments=['l', 'r']):
```

Comment:
Mutable default values are often problematic: If something does mutate them, you're in trouble.

Reply:
Fixed.